packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j-shared.sh

```diff
@@ -88,6 +88,10 @@ _find_java_cmd() {
 
   if [[ "${JAVA_HOME:-}" ]] ; then
     JAVA_CMD="${JAVA_HOME}/bin/java"
+    if [[ ! -e "${JAVA_CMD}" ]]; then
```

Comment:
Maybe it should be `! -f` even? Finding a folder called java won't help us. It's unlikely, but if their path is wrong, who knows how wrong it can be.